Implement multi-room chat

kiliman opened the pull request.

Commit:
Implement multi-room chat

davidfowl commented on Chat/Chat.cs:
```diff
@@ -155,7 +159,7 @@ namespace SignalR.Samples.Hubs.Chat
             }
 
-            content = Sanitizer.GetSafeHtmlFragment(content);
+            content = HttpUtility.HtmlEncode(content); //Sanitizer.GetSafeHtmlFragment(content);
```
Nope. Put that back in there.

davidfowl commented on Chat/Chat.js:
```diff
@@ -487,12 +539,78 @@ $(function () {
+        $('#messages-' + roomId).addClass('current').show();
+        $('#users-' + roomId).addClass('current').show();
+
+        if (roomId == "lobby") {
```
#

davidfowl replied:
`===`

davidfowl commented on Chat/Chat.js:
```diff
@@ -217,21 +230,34 @@ $(function () {
+        };
+
+        var room = user.Room;
+        if (!room) return;
```
No self closing ifs.

Commit:
Made changes based on code review

- Replace Sanitizer.GetSafeHtmlFragment()
- No more empty if blocks
- Use === and !==
- Add note about mobile version
